### DISCUSSION TITLE
Does PiJuice only work with Pijuice batteries? Stuck on a project

### DISCUSSION THREAD

**Discussion by Batselot** [PiSupply/PiJuice#762] [Q&A]
Hi there all,

I am working on a project that requires for me to work with 12000 mAH Pijuice battery, but it seems they are either out of stock or too long to ship ( because they are dangerous and cannot be shipped by airways). So the main question here is , can I use only PiJuice batteries to use them to full functionality? Or is there a way to use 3.7 12000 mAH batteries with some configuration? If so can someone give me a guide? 

I am very new to these projects , so I appreciate any kind of help.

**Comment by tvoverbeek** [comment]
You can use any single cell LiPo battery.
From the README on the hardware page (https://github.com/PiSupply/PiJuice/tree/master/Hardware#top-of-the-board):

The screw terminal can be used to connect an off-board / external battery instead of the provided onboard battery (the BP7X). We recommend you use a battery with an NTC thermistor integrated with the battery and used for temperature regulation of battery during charging we also recommend that the battery have onboard over charge / over current etc. protection (this is not essential, but it is far safer if you have them - lithium batteries can be dangerous so it is wise to think about safety). The requirements are that battery uses a 10KOhm NTC resistor with known thermistor B constant which can be entered as profile data in config GUI "Battery" tab. There are regulation threshold data points that can be entered for custom batteries like Cold, Cool, Warm and HOT temperatures that are derived from the battery manufacturers specification. The ID pin is not used and it is just reserved for possible future use. It could for example be used to automatically recognise which battery is connected when using BP7X or BP6X. The screw terminal is intentionally facing inwards, towards the onboard battery, for safety...this is not a mistake! The idea is to make it harder for you to accidentally plug in two batteries at once and fry your PiJuice. If you would like to turn the screw terminal to face the other direction you can do so by unscrewing all four screws, turning it around, and then screwing them back in again. Please note that you do this at your own risk and it will void your warranty.
